cmd/kubelet/app/server.go

```diff
@@ -644,11 +643,11 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.Dependencies, stopCh <-chan
 			klog.Info("--cgroups-per-qos enabled, but --cgroup-root was not specified.  defaulting to /")
 			s.CgroupRoot = "/"
 		}
-		kubeReserved, err := parseResourceList(s.KubeReserved)
+		kubeReserved, err := helper.ParseResourceList(s.KubeReserved)
 		if err != nil {
 			return err
 		}
-		systemReserved, err := parseResourceList(s.SystemReserved)
+		systemReserved, err := helper.ParseResourceList(s.SystemReserved)
```

Comment:
If we have validated the params in `kubeletconfigvalidation.ValidateKubeletConfiguration`, is it necessary to validate repeatedly here ?

Reply:
It appears the validation is not necessary, rather the parsing into a `ResourceList`, as it is used by the `ContainerManager` in `server.go`. I don't know if we should separate the parsing and validating into two different functions.